adam/language_specific/english/english_language_generator.py

```diff
@@ -156,7 +191,7 @@ def _translate_action_to_verb(
             # TODO: we don't currently handle verbal morphology.
             # https://github.com/isi-vista/adam/issues/60
             verb_dependency_node = DependencyTreeToken(
-                lexicon_entry.verb_form_3SG_PRS, lexicon_entry.part_of_speech
+                str(lexicon_entry.verb_form_3SG_PRS), lexicon_entry.part_of_speech
```

Comment:
Do you need `str` here because it was complaining about it being `Optional[str]`? What you want to do instea dis to explicitly check it for being `None` and throw an exception if it is

Reply:
Done